packages/blade/src/components/Card/CardSurface.native.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import { Platform } from 'react-native';
+import styled from 'styled-components/native';
+import { useTheme } from '~components/BladeProvider';
+import Box from '~components/Box';
+import type { BoxProps } from '~components/Box/types';
+
+const isAndroid = Platform.OS === 'android';
+// TODO: Temporary workaround to make android shadows look as close as iOS
+const androidShadow = {
+  color: undefined,
+  elevation: 2,
```

Comment:
Lets use our elevation tokens here?

https://github.com/razorpay/blade/blob/master/packages/blade/src/tokens/theme/paymentTheme.ts#L2192

Reply:
Need to do this for android to make it look close to iOS shadow. Can't use token here since in the value is 5 there. :(

https://razorpay.slack.com/archives/G01B3LQ9H0W/p1671601580827619?thread_ts=1671538782.461409&cid=G01B3LQ9H0W

Reply:
We need to relook at our elevation tokens then. Lets hardcode it now but can you raise an issue to track this?

Reply:
Yes raised issue already #922